org.eclipse.scanning.test/src/org/eclipse/scanning/test/points/SpiralTest.java

```diff
@@ -52,9 +52,9 @@ public void testSpiralNoROI() throws Exception {
 
 		// Test a few points
 		// TODO check x and y index values - currently these are not tested by AbstractPosition.equals()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)